modules/storage/cosmos-db/version.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://aka.ms/bicep-registry-module-version-file-schema#",
-  "version": "1.0",
+  "version": "2.0",
```

Comment:
@sunilthorat09 my last nitpick here. I don't think we actually broke anything with this update, so you could do 1.1 instead. But, since this is such a big update, I'm also fine with staying at 2.0.

Reply:
I think there are changes to parameters that makes the major version bump necessary (e.g., a couple of parameters got removed).